odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -50,27 +196,44 @@ def fbp_op(ray_trafo, padding=True):
         `CircularConeFlatGeometry` : Approximate reconstruction, correct in
         limit of fan angle = 0 and cone angle = 0.
 
-        `HelicalConeFlatGeometry` : Very approximate.
+        `HelicalConeFlatGeometry` : Very approximate unless a
+        `tam_danielson_window` is used. Accurate with the window.
```

Comment:
Thanks for the explanation, I was just curious, no need to go into extreme detail in the doc. But to me this windowing seems like a cheap trick to get rid of unwanted influences in the data, at the cost of either artefacts from the jump or inexactness due to the smoothing. And I guess noise doesn't improve the situation.. Anyway, we can close this chapter.